server/index.js

```diff
@@ -1,16 +1,29 @@
 const express = require('express');
+const path = require('path');
 let app = express();
-let { retrieve, seeder } = require('./db/index.js')
+let bodyParser = require ('body-parser');
+let { retrieve } = require('./db/index.js')
+let port = 4000;
 
 app.use(express.static(__dirname + '/../client/dist'));
 
-app.get('/', function (req, res) {
-  console.log('Got request');
-  seeder();
-  res.end();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.get('/buildings/:id', function (req, res) {
+  console.log(`Got request for ${req.params.id}`);
+  let index = path.resolve(__dirname + '/../client/dist/index.html');
+  let options = { id: req.params.id };
+  res.sendFile(index, options);
 });
 
-app.get('/show', function (req, res) {
+app.get('/workspace-api/get/:id', function (req, res) {
```

Comment:
A common practice with rest APIs is to use nouns instead of verbs for the routes. For that reason, I would write this route as `/workspace-api/workspace/:id` and the get all route as `/workspace-api/workspaces`.

Reply:
Thanks for the info on that, I'll change that now.